### PR TITLE
use attributes from core to identify anamorphic videos

### PR DESCRIPTION
**Pull request by mistermendez**
⚠️ must be merged with - https://git.corp.ooyala.com/projects/PBW/repos/mjolnir/pull-requests/186/overview

[sandbox test link](http://debug.ooyala.com/ea/index.html?ec=p3eWQ1ODE6kGcyoYZj8tNOnPwEQJKxvc&pbid=ebcd684d5516464f84b6eae21bdb1328&pcode=8yMGYyOvhG5HuuITgXl_UUwes_5e&core_player=http%3A%2F%2Fplayer.ooyala.com%2Fstatic%2Fv4%2Fsandbox%2Fanamorphic%2Fcore.js&video_plugins=http%3A%2F%2Fplayer.ooyala.com%2Fstatic%2Fv4%2Fcandidate%2Flatest%2Fvideo-plugin%2Fbit_wrapper.min.js&html5_skin=http%3A%2F%2Fplayer.ooyala.com%2Fstatic%2Fv4%2Fsandbox%2Fanamorphic%2Fskin-plugin%2Fhtml5-skin.js&skin_asset=http%3A%2F%2Fplayer.ooyala.com%2Fstatic%2Fv4%2Fsandbox%2Fanamorphic%2Fskin-plugin%2Fhtml5-skin.css&additional_plugins=http%3A%2F%2Fplayer.ooyala.com%2Fstatic%2Fv4%2Fcandidate%2Flatest%2Fother-plugin%2Fdiscovery_api.min.js&options=%7B%22autoplay%22%3Afalse%2C%22initialBitrate%22%3A%7B%22level%22%3A%221%22%2C%22duration%22%3A%225%22%7D%2C%22debug%22%3Atrue%2C%22embedToken%22%3A%22http%3A%2F%2Fplayer.ooyala.com%2Fsas%2Fembed_token%2F8yMGYyOvhG5HuuITgXl_UUwes_5e%2Fp3eWQ1ODE6kGcyoYZj8tNOnPwEQJKxvc%3Faccount_id%3Dmax%26api_key%3D8yMGYyOvhG5HuuITgXl_UUwes_5e.9u6Cq%26expires%3D1482252214%26signature%3DtN1rnn%2FOfDBn0kUDs2IYzGQS3jD5noNZRu0l2sgLRVU%22%7D)

### CODE REVIEW

File: js/controller.js [hunk 1]
```diff
@@ -233,6 +235,9 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.state.mainVideoContainer.addClass('oo-player-container');
       this.state.mainVideoInnerWrapper.addClass('oo-player');
       this.state.mainVideoInnerWrapper.append("<div class='oo-player-skin'></div>");
+      if (this.state.playerParam.anamorphic == true || this.state.playerParam.anamorphic == "true") {
```
Comment (mistermendez) [inline js/controller.js]: this page-level param is added as a fallback until PBS API updates go live

**Comment by mistermendez** [comment]
FYI, code review needed @jpfranco or @pilievOoyala

File: scss/skin/_default.scss [hunk 2]
```diff
@@ -67,6 +67,11 @@ video {
   @extend .oo-expand;
 }
 
+// fix anamorphic video issue on Firefox
+@include apply-to-firefox-only ('.oo-anamorphic video') {
+  object-fit: fill !important;
```
Comment (jpfranco) [inline scss/skin/_default.scss]: Will this work in fullscreen as well, or will it distort the aspect ratio by filling up the viewport?

**Comment by jpfranco** [comment]
Sorry @mistermendez  I'm just seeing this now. I didn't get notified for some reason.

File: js/controller.js [hunk 3]
```diff
@@ -1084,6 +1096,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.mb.unsubscribe(OO.EVENTS.CLOSED_CAPTIONS_INFO_AVAILABLE, "customerUi");
       this.mb.unsubscribe(OO.EVENTS.BITRATE_INFO_AVAILABLE, "customerUi");
       this.mb.unsubscribe(OO.EVENTS.CLOSED_CAPTION_CUE_CHANGED, "customerUi");
+      this.mb.unsubscribe(OO.EVENTS.CHANGE_CLOSED_CAPTION_LANGUAGE, "customerUi");
```
Comment (pilievOoyala) [inline js/controller.js]: Is this something that was missed before? Doesn't seem related to attributes.

Comment (mistermendez) [inline js/controller.js]: yea, i noticed it was missing

File: scss/skin/_default.scss (hunk 2 above)
Comment (mistermendez) [inline scss/skin/_default.scss]: will test..

Comment (mistermendez) [inline scss/skin/_default.scss]: correct aspect ratio maintained in fullscreen as well -

screen shot 2017-02-06 at 1 02 20 pm

screen shot 2017-02-06 at 1 02 25 pm

**Comment by jpfranco** [comment]
@pilievOoyala LGTM if it LGTY

**Comment by pilievOoyala** [comment]
*thumbs up*
